src/open_inwoner/templates/pages/cases/status.html

```diff
@@ -42,7 +42,7 @@ <h2 class="h2" id=>{% trans "Document toevoegen" %}</h2>
                 {% elif case.external_upload_enabled %}
                     <p class="p">{% trans "By clicking the button below you can upload a document. This is an external link and you will be redirected to a different system." %}</p>
                     {% button_row %}
-                        {% button href=case.external_upload_url text=_("Document toevoegen") primary=True %}
+                        {% button href=case.external_upload_url text=_("Document toevoegen") primary=True icon="open_in_new" icon_position="after" %}
```

Comment:
For accessibility purposes the icon here needs an aria-label with the translatable text "Opens new window" - but it's also OK to put the aria-label in the Anchor link if that's easier.